File: TsogosunProfileAdminWeb/src/app/model/request/profile-rewards-admin/requestAddUpdateLeaderBoard.ts
import { LeaderBoardPromotionDto } from "../../Dtos/profile-rewards-admin/leaderBoardPromotionDto";

export interface RequestAddUpdateLeaderBoard {
    startTime?: string | undefined;
    endTime?: string | undefined;
    leaderBoardPromotionDto?: LeaderBoardPromotionDto | undefined;
}

File: TsogosunProfileAdmin/MSPatronRewardsAdmin/Model/Request/RequestAddUpdateLeaderBoard.cs


using MSPatronRewardsAdmin.Model.Dtos;

namespace MSPatronRewardsAdmin.Model.Request
{
    public class RequestAddUpdateLeaderBoard 
    {
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public LeaderBoardPromotionDto LeaderBoardPromotionDto { get; set; }

    }
}
